javascript-methods/main.js

```diff
@@ -0,0 +1,53 @@
+var first = 1;
+var second = 2;
+var third = 3;
+var maximumValue = Math.max(first, second, third);
+console.log('the maximum value is:', maximumValue);
+var heroes = ['Captain America', 'vision', 'ironman', 'Black Panther'];
+var randomNumber = Math.random();
+randomNumber = heroes.length * randomNumber;
+var randomIndex = Math.floor(randomNumber);
+console.log('randomIndex value:', randomIndex);
+var randomHero = heroes[randomIndex];
+console.log('randomHeroes value:', randomHero);
+
+var library = [
+  {
+    title: 'Ender-Shadow',
+    author: 'Orson Scott Card'
+  },
+  {
+    title: 'Harry Potter',
+    author: 'JK Rowling'
+  },
+  {
+    title: 'Enders-Game',
+    author: 'Orson Scott Card'
+  }
+];
+var lastBook = library.pop();
+console.log('lastBook value:', lastBook);
+var firstBook = library.shift();
+console.log('value of firstBook:', firstBook);
+
+var js = {
+  title: 'Javascript for Impatient Programmers',
+  author: 'Dr.Axel Rauschmayer'
+};
+
+var css = {
+  title: 'CSS Secrets',
+  author: 'Lea Verou'
+};
+
+library.push(js);
+library.unshift(css);
+library.splice(1);
```

Comment:
you are missing the second argument here which tells splice how many items to remove. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice